packages/create-platformatic/src/db/create-db.mjs

```diff
@@ -136,7 +136,7 @@ async function createDB ({ hostname, database = 'sqlite', port, migrations = 'mi
     logger.info('Configuration file platformatic.db.json successfully created.')
 
     const env = generateEnv(hostname, port, database)
-    await writeFile(join(currentDir, '.env'), env)
+    await appendFile(join(currentDir, '.env'), env)
```

Comment:
If the file does not have a newline at the end, this will not work well. It't better to read the content first

Reply:
I tested this and is does not seem to have any problem with the new line as the `generateEnv` returns this string

```js
  const env = `\
PLT_SERVER_HOSTNAME=${hostname}
PORT=${port}
PLT_SERVER_LOGGER_LEVEL=info
DATABASE_URL=${connectionString}
`
  return env
```

Reply:
Do you still find it important to read the file first?

Reply:
no this is ok